examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
```

Comment:
Yes, don't do it this way if you want jenkins tests to pass in their current form (the docker command line has a reference to a server in the openshift.com domain).